lib/dialects/abstract/query-generator.js

```diff
@@ -2149,7 +2149,7 @@ var QueryGenerator = {
   },
 
   geometrySelect: function(column) {
-    return this.quoteIdentifiers(column);
+    return 'AsText(' + this.quoteIdentifiers(column) + ') AS ' + column;
```

Comment:
I'm pretty sure sqlite uses `AsText` and `GeomFromText` as well

There is a node module for spatialite (the GIS version of sqlite). But it doesn't seem to be very well maintained https://github.com/zhm/node-spatialite